go-packages/meep-redis/db.go

```diff
@@ -41,11 +43,20 @@ type Connector struct {
 // NewConnector - Creates and initialize a Redis connector
 func NewConnector(addr string, table int) (rc *Connector, err error) {
 	rc = new(Connector)
-	err = rc.connectDB(addr, table)
+
+	// Connect to Redis DB
+	for retry := 0; !rc.connected && retry <= dbMaxRetryCount; retry++ {
+		err = rc.connectDB(addr, table)
+		if err != nil {
+			log.Warn("Failed to connect to DB. Retrying... Error: ", err)
+			continue
```

Comment:
useless code

Reply:
This logs the error cause before retrying.

Reply:
WRT continue I agree that it could be removed here without impacting the code. Will remove it.